release_notes/topics/18_0_0.adoc

```diff
@@ -31,6 +31,12 @@ Thanks to https://github.com/vanrar68[Joaquim Fellmann] for the contribution.
 
 There are more WebAuthn improvements and fixes in addition to that.
 
+== Client secret rotation
+
+{project_name} now supports Client Secret Rotation through customer policies. This feature allows thats confidential clients can be provided with realm policies allowing the use up to two secrets simultaneously.
```

Comment:
Is it possible to add note that this is preview feature? See "Recovery Codes" section for the inspiration :-)

Reply:
Done